iscript/src/iscript/mac.py

```diff
@@ -279,11 +293,13 @@ async def sign_app(sign_config, app_path, entitlements_path, provisioning_profil
 
         for file_ in files:
             abs_file = os.path.join(top_dir, file_)
+            sign_command = _get_sign_command(identity, keychain, sign_config, file_, entitlements_path)
```

Comment:
Nit: I tend to prefer explicitly setting kwargs, e.g. `file_=file_`. Non-blocking if it passes the linter.